plugins/outputs/riemann/riemann.go

```diff
@@ -68,8 +79,16 @@ func (r *Riemann) Write(metrics []telegraf.Metric) error {
 
 	var senderr = r.client.SendMulti(events)
 	if senderr != nil {
```

Comment:
great, thanks for clarification.
should I squash PR into one commit, if it will be accepted for merge?